src/main/python/ayab/preferences.py

```diff
@@ -219,20 +244,20 @@ class PrefsComboWidget(QComboBox):
     @author Tom Price
     @date   July 2020
     """
-    def __init__(self, prefs, var):
+    def __init__(self, prefs:Preferences, var:PreferencesDictObjKeys):
         super().__init__()
         self.var = var
         self.prefs = prefs
         cls = self.prefs.variables[self.var]
         cls.add_items(self)
```

Comment:
this is actually really slick- all of the values with keys in `PreferencesDictObjKeys` implement a `add_items` method, so the typing just _works_.